### PR TITLE
Bitwarden: Actually print an error message if no result was found

### PR DESCRIPTION
**Pull request by MarcelBochtler**
## Description

Notify the user if the searched password was not found.

## Type of change

- [ ] New script command
- [x] Bug fix
- [x] Improvement of an existing script
- [ ] Documentation update
- [ ] Toolkit change
- [ ] Other (Specify)

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/password-managers/bitwarden/copy-first-matching-password.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@
 # @raycast.description Search all items in a Bitwarden vault, and copy the password of the first search result to the clipboard.
 
 notFound() {
-  echo "The query '$1' did not return a password."
+  echo "The query '${BASH_ARGV[0]}' did not return a password."
```
Comment (PSalant726) [inline commands/password-managers/bitwarden/copy-first-matching-password.sh:25]: What does this change accomplish?

**Review by PSalant726 — CHANGES_REQUESTED** [review]
One question, but thanks for fixing the bad ref!

File: commands/password-managers/bitwarden/copy-first-matching-password.sh (hunk 1 above)
Comment (MarcelBochtler) [reply commands/password-managers/bitwarden/copy-first-matching-password.sh:25]: `$1` in the function is the first parameter passed to the _function_ and not the first parameter passed to the _script_.
`${BASH_ARGV[0]}` is the first parameter of the _script_ itself, which is the intended value here.

As an alternative we could call `notFound` with the parameter `$1` => `name=$(echo $item | jq --exit-status ".name") || notFound $1`
What do you prefer?

Comment (PSalant726) [reply commands/password-managers/bitwarden/copy-first-matching-password.sh:25]: Ah, of course. That makes sense. I prefer what you have here.

**Review by PSalant726 — APPROVED** [review]
Thanks for the changes!

**Pull request merged**